.github/workflows/build-distributions.yml

```diff
@@ -0,0 +1,48 @@
+name: Build installable distrabutions
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build-distribution:
+    strategy:
+      matrix:
+        os: [ macos-latest ] #windows-latest, ubuntu-latest,
+        runner: [
+          {
+            name: arelleCmdLine,
+            executor: arelleCmdLine.py
+          },
+          {
+            name: arelleGUI,
+            executor: arelleGUI.pyw
+          }
+        ]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install Python 3
+        uses: actions/setup-python@v3
+        with:
+          python-version: 3.9
+      - run: pip install -r requirements.txt pyinstaller
```

Comment:
Move pyinstaller into requirements (and version it) where cx_freeze is defined

Reply:
I don't feel strongly, however, its not really a requirement and there isn't really a way to test it when you update it so idk if we should be updating with dependabot

Reply:
I feel strongly about this. Build requirements are dev requirements IMO (folks should be able to build for their platform locally). More importantly, I don't want to have to update/manage dependency versions in github workflows for each build if we have to lock the version of pyinstaller at some point because of a breaking change.